### DISCUSSION TITLE
Show Reset information in Last Activities

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#443] [Q&A] [answered]
Hey,

I'm currently looking to optimise my sync process and I'm stuck on one thing. When I reset the progress for a show, I'm not finding a good way to get it fast enough (with Last Activities).

Currently, I'm relying on `/sync/last_activities` for a lot of things like show hidden from progress and last watched at info to trigger a sync but the reset doesn't trigger a sync because there doesn't seem to be a Last Activity linked to the reset action.

Would it be possible to add that or maybe there's another endpoint I can look at?

Cheers,

Kevin

**Comment by tysonkerridge** [comment, marked as answer]
Unless it's bugged in some way, I believe it should be last activities `shows.hidden_at` that changes when reset items change. That one seems to encapsulate most of the "hidden" path endpoints.

**Reply by kevincador** [reply]
Perfect, I can confirm it works like that! I always forget the reset is linked to the hidden stuff.
Thanks a lot @tysonkerridge
